.github/workflows/release.yml

```diff
@@ -367,19 +423,32 @@ jobs:
     - name: Create download folder
       run: |
         mkdir -p ${{ matrix.os }}-${{ matrix.arch }}-bin
+        mkdir -p ubuntu-tracing-bin
 
     - name: Download pre-built collator binary
       uses: actions/download-artifact@v3
       with:
         name: astar-${{ matrix.os }}-latest-${{ matrix.arch }}
         path: ${{ matrix.os }}-${{ matrix.arch }}-bin
 
+    - name: Download pre-built collator binary for evm tracing
```

Comment:
Download & Upload EVM tracing binary should be separate task.
Otherwise, they are called 3 times because matrix is defined as a strategy.
```
strategy:
      matrix:
        os: ["ubuntu", "macOS"]
        arch: ["x86_64", "aarch64"]
        exclude:
        - os: macOS
          arch: aarch64
```